epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityItem.java

```diff
@@ -139,11 +145,15 @@ boolean handleChanged(EpoxyViewHolder epoxyHolder, boolean visibilityChangedEnab
     boolean changed = false;
     if (visibleHeight != lastVisibleHeightNotified || visibleWidth != lastVisibleWidthNotified) {
       if (visibilityChangedEnabled) {
-        epoxyHolder.visibilityChanged(
-            100.f / height * visibleHeight,
-            100.f / width * visibleWidth,
-            visibleHeight, visibleWidth
-        );
+        if (viewVisibility == View.GONE) {
+          epoxyHolder.visibilityChanged(0f, 0f, 0, 0);
```

Comment:
do we need to check that `viewVisibility` changed? could it have already been gone but had the size change?

Reply:
Nice catch! I'd agree it makes sense to also check that. I'll make that change.

Reply:
@elihart While testing this I realized the size won't change while it's gone since the layout pass won't re-measure. I did find a case that makes us need this last visibility notified check though. If the view transitions from gone to visible the size won't change so we would need that check. I've added it and this should be good to go now.